src/EmailReplyParser/Parser/EmailParser.php

```diff
@@ -21,7 +21,7 @@ class EmailParser
     /**
      * Regex to match signatures
      */
-    const SIG_REGEX   = '/(?:^\s*--|^\s*__|^-\w|^-- $)|(?:^Sent from my (?:\s*\w+){1,3})$/s';
+    const SIG_REGEX   = '/(?:^\s*--|^\s*__|^-\w|^-- $)|(?:^(Sent from my)|(发自我的) (?:\s*\w+){1,3})|(?:^나의(?:\s*\w*)(에서\s보냄)|(から送信))$/s';
```

Comment:
Well spotted.
A test was created.
I restored the original SIG_REGEX since I wasn't able to get the tests to pass.
Systems will just have to suffer and see the "Sent from my iPhone" in multilingual versions a little bit longer.